.github/workflows/main.yml

```diff
@@ -51,4 +56,13 @@ jobs:
         pactl load-module module-remap-source master=virtmic.monitor source_name=virtmic source_properties=device.description=Virtual_Microphone
 
         # test stuff
-        python3 -m pytest tests
\ No newline at end of file
+        python3 -m pytest --cov=./src/iSparrowRecord --cov-report=xml
+
+    - name: Upload coverage reports to Codecov
+      uses: codecov/codecov-action@v4
```

Comment:
could add here that report is only uploaded for latest Python version

Reply:
add the python version to the name. that ok?